sbnci/Modules/RecoEff/reco_eff.fcl

```diff
@@ -0,0 +1,44 @@
+#include "simulationservices_sbnd.fcl"
+#include "backtrackerservice.fcl"
+#include "particleinventoryservice.fcl"
+
+#include "analysistreemodule.fcl"
+#include "rootoutput_sbnd.fcl"
+#include "showerfindermodules_sbnd.fcl"
+
+process_name: AnaTree
+
+services:
+{
+	FileCatalogMetadata      : @local::sbnd_file_catalog_mc
+	TFileService             : { fileName: @local::sbnd_rootoutput.fileName }
+	@table::sbnd_services
+	ParticleInventoryService : @local::standard_particleinventoryservice
+	BackTrackerService       : @local::standard_backtrackerservice
```

Comment:
are these not already setup as part of the sbnd_services table?

Reply:
I don't think so?